plugin/resolvergen/resolver.go

```diff
@@ -68,16 +68,17 @@ func (m *Plugin) generateSingleFile(data *codegen.Data) error {
 				continue
 			}
 
-			resolver := Resolver{o, f, nil, "// foo", `panic("not implemented")`}
```

Comment:
Currently, a fixed comment `// // foo` is generated for all resolvers in `single-file` mode, but it appears to me as if it was accidentally added to the code during development and was not intentional. Therefore, I have made a modification to generate the same comments as in `follow-schema` mode.